src/main/app/form/validation.ts

```diff
@@ -106,8 +108,29 @@ export const isPhoneNoValid: Validator = value => {
   }
 };
 
+export const isEmailValid: Validator = value => {
+  if (!EmailValidator.validate(value as string)) {
```

Comment:
Then, this could be:
```suggestion
  if (!isValidEmail(value as string)) {
```